src/components/HourGuideCell.tsx

```diff
@@ -28,7 +39,7 @@ export const HourGuideCell = ({ cellHeight, onPress, date, hour, index }: HourGu
           u['border-b'],
           { borderColor: theme.palette.gray['200'] },
           { height: cellHeight },
-          { backgroundColor: isPair(index) ? evenCellBg : oddCellBg },
```

Comment:
I think `isPair` function can be deleted safely (from util.ts)

Reply:
@TechSaq I am not able to link this project locally for adding / fixing an issue, can you share how you did it ?

Reply:
@iranaahsanali Can you please share what issue are you getting? Please share your system details also. I also got a issue due to different Node version for my Windows 10 system. If you are also having the same issue, I will be happy to help.

Reply:
Hi, @TechSaq @acro5piano   Actually my requirements were a little bit different from this repo, so I forked this repo, and made some changes on it, I was testing the changes through storybook build, So after making changes I pushed that code to my repo. So when I installed the package in my project from my forked repo, it is throwing an error, main module was not resolved. Please check the screenshot.

![Simulator Screen Shot - iPad (9th generation) - 2022-01-15 at 22 06 01](https://user-images.githubusercontent.com/18231674/149630807-3a36cd4a-3932-4bd8-b8aa-f621e6e0a23a.png)

Reply:
@iranaahsanali That is off-topic. Please create a new issue instead. Thanks